docs/glossary.md

```diff
@@ -15,6 +15,8 @@ To help with clarity and a common understanding, here are some helpful terms and
 
 Chains in the superchain-registry are assigned a `superchain_level` (shown in individual config files as well as the `chainList.toml/json` summaries), depending on the set of validation checks that they pass.
 
-**Frontier** chains have `superchain_level = 0`. These are members of the Superchain ecosystem.
+**Non-Standard** chains have `superchain_level = 0`. These are members of the Superchain ecosystem.
```

Comment:
What does "member of the Superchain ecosystem" mean? Would it be accurate and less ambiguous to say: "These are chains that run on the OP stack, but do not conform to the standard blockchain charter"